examples/Alert.sticky.tsx

```diff
@@ -7,7 +7,14 @@ import { Alert } from '@itwin/itwinui-react';
 
 export default () => {
   return (
-    <div>
+    <div
+      style={{
+        height: '150px',
+        overflow: 'overlay',
+        textAlign: 'justify',
+        border: 'solid 0.5px',
+      }}
+    >
```

Comment:
`overflow: overlay` is deprecated and `justify` alignment should never be used

```suggestion
    <div style={{ height: 150, overflow: 'auto', border: 'solid 0.5px' }}>
```